packages/common/src/core/getEntityDefaultWorkspacePane.ts

```diff
@@ -1,6 +0,0 @@
-export const getEntityDefaultWorkspacePane = (entityType: string): string => {
```

Comment:
Technically removing this util is a breaking change - I think what we should do for now is just add a deprecation notice letting users know something like:
1. This util is a duplicate and concern of the UI, and was moved as such
2. We will be removing it in the next breaking version

Reply:
Good call -- am I able to stop exporting it from `core/index.ts`, or would that cause a breaking change as well?

Reply:
yeah, I think that would still be considered a breaking change since there _could_ be some other team importing this util - obviously that is not likely/almost certainly not happening...but I do think we still have to consider it a breaking change?